tests/dummy/app/templates/application.hbs

```diff
@@ -65,7 +65,7 @@
             <li>
               <label>lat/lng: {{input value=entry.lat class='coord'}}
                 /{{input value=entry.lng class='coord'}}</label>
-              <button {{action 'removePolylinePathItem' path entry}}>x</button>
+              <button {{action 'removePolylinePathItem' polyline.path entry}}>x</button>
```

Comment:
Thank you! Can you check that this is not also the case for polygons and fix it there also in that case? Sorry can't now, on my phone.